app/assets/javascripts/sidebar.js

```diff
@@ -0,0 +1,20 @@
+$(function () {
+  // Dynamically setup position of sidebar handles.
+  function positionMultipleSidebars() {
+    var sidebarTop = $('.sidebar-mod').offset().top;
+    var sidebarHdrHeight = $('.sidebar-hdr').height() + 35;
```

Comment:
Thanks, hopefully improved in https://github.com/concord-consortium/lara/commit/37cb47c448687300cb8cec63d3b67fbabbe320d3.